README.md

```diff
@@ -1,7 +1,7 @@
 
-# Jellyfish Android Client
+# Fishjam Android Client
 
-Android client library for [Jellyfish](https://github.com/jellyfish-dev/jellyfish).
+Android client library for [Fishjam](https://github.com/jellyfish-dev/jellyfish).
```

Comment:
Shouldn't we update the links to point to the new org and repo as well?

Reply:
yesterday new names were not available, but right now it can be updated too.

Reply:
From today links can be updated safely, i will do it

Reply:
As we discussed yesterday repo code will be updated later.